### ISSUE TITLE
Site Editor: What access level to give site admins?

### ISSUE THREAD

**Issue by knice** [ucsc/theme-ucsc#13]
Will site admins have access to the site editor? 

## What are the implications for theme updates if site admins can make **local** theme file changes?
- Local theme file adjustments might not get updated theme template changes
- Reverting local theme file edits will result in changes

**Comment by knice** [comment]
**We cannot give site owners access to modify the theme** within the theme editor because:

1. Local theme changes take precedence over theme file changes. If we update the theme files and release them to a site, the updates won't take effect on the site because of a local theme edit.
2. If the site owner clears their local theme edits so an updated theme file can take effect on their site, they lose ALL customizations they made to that file.

In short, allowing site owners to make local theme modifications prevents us from updating their theme template files in future releases of the theme.
